app/kubemci/pkg/gcp/loadbalancer/loadbalancersyncer.go

```diff
@@ -173,12 +173,18 @@ func (l *LoadBalancerSyncer) CreateLoadBalancer(ing *v1beta1.Ingress, forceUpdat
 			err = multierror.Append(err, cErr)
```

Comment:
You can remove the TODO for HTTPS and the ERROR about HTTPS?

Reply:
Good catch. Done